org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/BatchCompilerTest.java

```diff
@@ -11325,7 +11277,7 @@ public void testInferenceIn14Project(){
 	}
 }
 // https://bugs.eclipse.org/bugs/show_bug.cgi?id=328775 - Compiler fails to warn about invalid cast in 1.4 mode.
-public void testInferenceIn15Project(){  // ensure 1.5 complains too
+public void testInferenceIn15Project(){  // ensure 1.8 complains too
```

Comment:
This test and the previous one testInferenceIn14Project are testing the same thing I think

Reply:
Correct. I've removed testInferenceIn14Project